src/index.js

```diff
@@ -107,6 +108,13 @@ VirtualMachine.prototype.postIOData = function (device, data) {
  * @param {string} json JSON string representing the project.
  */
 VirtualMachine.prototype.loadProject = function (json) {
+    if (!json) {
```

Comment:
I'd still prefer it if this were outside of `loadProject`, before/after it's called. The reason being that the description of this function - "Load a project from a Scratch 2.0 JSON representation." - says nothing about loading a fallback.